healthcheck/dial.go

```diff
@@ -32,6 +30,7 @@ type conn struct {
 	fd int
```

Comment:
I didn't check carefully. Are these two fields still needed?

Reply:
After further check, I remove these fields since they are not used any more